nuget and msbuild errors when creating building the Dockerfile.

<!--- This template provides sections for bugs and features. Please delete any irrelevant sections before submitting -->

#### Expected Behavior
<!--- Required. Describe the behavior you expect to see for your case. -->
Build without any nuget or msbuild issue.

#### Actual Behavior
<!--- Required. Describe the actual behavior for your case. -->
I get the following error
```
The 'Newtonsoft.Json 10.0.3' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.60717.93'.
'QLNet' already has a dependency defined for 'System.Reflection.Emit.Lightweight'.
The 'System.Net.Http 4.3.3' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.60717.93'.
The 'Microsoft.CSharp 4.6.0' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.60717.93'.
The 'System.Reflection.Emit 4.3.0' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.60717.93'.
The 'System.Reflection.Emit.Lightweight 4.3.0' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.60717.93'.
The 'System.ValueTuple 4.4.0' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.60717.93'.
```

After upgrading the nuget version, I get the following msbuild error
```
MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains more than one project or solution file.
```

#### Potential Solution
<!--- Optional. Describe any potential solutions and/or thoughts as to what may be causing the difference between expected and actual behavior. -->
Downloading latest nuget and specif
#### Reproducing the Problem
<!--- Required for Bugs. Describe how to reproduce the problem. This can be via a failing unit test or a simplified algorithm that reliably demonstrates this issue.  -->
Upgrade the Nuget version then specify the solutions file when calling msbuild.

#### System Information
<!--- Required for Bugs. Include any system specific information, such as OS. -->

#### Checklist
<!--- Confirm that you've provided all the required information. -->
<!--- Required fields --->
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)
<!--- Required for Bugs, feature request can delete the line below. -->
- [x] I have provided detailed steps to reproduce the issue

<!--- Template inspired by https://github.com/stevemao/github-issue-templates -->

--- a/Dockerfile
+++ b/Dockerfile
@@ -16,9 +16,9 @@ MAINTAINER QuantConnect <contact@quantconnect.com>
 # RUN \
 #     cd /root/Lean && \
 #     sed -i 's/4.5/4.0/' Algorithm.VisualBasic/QuantConnect.Algorithm.VisualBasic.vbproj && \
-#     wget https://nuget.org/nuget.exe && \
+#     wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe && \
 #     mono nuget.exe restore QuantConnect.Lean.sln -NonInteractive && \
-#     msbuild /property:Configuration=Release && \
+#     msbuild QuantConnect.Lean.sln /property:Configuration=Release && \
 #     cd /root/Lean/Launcher/bin/Release/
 #################################
 

